evals/runners/run_mcp_vs_vanilla.py

```
@@ -751,13 +752,41 @@ def _winner_corner(winner: str) -> str:
     def _is_truncated(usage: dict[str, int]) -> bool:
         return arm_max > 0 and usage["output_tokens"] >= truncation_threshold
 
+    def _cat_winner(margin: float) -> str:
+        """Per-category Δ winner for badge colour — sign of (mcp − vanilla)."""
```

Ruff (0.15.14)

[warning] Docstring contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)